photon-client/src/mixins/global/dataHandleMixin.js

```diff
@@ -2,14 +2,14 @@ export const dataHandleMixin = {
     methods: {
         handleInput(key, value) {
             let msg = this.$msgPack.encode({[key]: value});
-            this.$socket.send(msg);
+            this.$store.state.websocket.send(msg);
```

Comment:
This can possibly throw a NPE right now lmao. But as soon as the app is created() it won't. Not sure if it's worth adding more error checking?

Reply:
nope :)